develop/learning-paths/themes/articles/01-setting-up-the-theme/04-modifying-theme-templates.markdown

```diff
@@ -0,0 +1,169 @@
+# Modifying Theme Templates
+
+In this section you'll add some finishing touches to `portal_normal.ftl` and
+`navigation.ftl` to match the finished mockup for the Lunar Resort.
+
+1. Open `portal_normal.ftl` and find the `<body class="${css_class}">` and 
+   replace it with the following code:
+   
+       <body class="${css_class} dockbar-split">
```

Comment:
This code isn't rendering; I think you need to indent it more.

I'm enjoying reading your learning path draft in preparation for sharing the Dev 1 slides on Themes. :+1: